docs/android/deployment/overview.md

```diff
@@ -13,133 +13,55 @@ ms.date: 03/17/2022
+- **Through a market** &ndash; There are several Android marketplaces that exist for distribution, such as  [Google Play](https://play.google.com/) or [Amazon App Store for Android](https://www.amazon.com/mobile-apps/b?ie=UTF8&node=2350149011).
 
-> [!TIP]
-> Some settings are available in the **Project Properties** editor in Visual Studio to change values. Right-click on the project in the **Solution Explorer** pane and choose **Properties**. For more information, see [Project configuration in .NET MAUI](../../deployment/visual-studio-properties.md).
+Using an established marketplace is the most common way to publish an app as it provides the broadest market reach and the greatest control over distribution. However, publishing an app through a marketplace requires extra effort.
 
-The following table describes how each project setting maps to the manifest file:
+Multiple channels can distribute a .NET MAUI app simultaneously. For example, an app could be published on Google Play, the Amazon App Store for Android, and also be downloaded from a web server.
 
-| Project setting | Manifest setting |
-| --- | --- |
-| `ApplicationId` | The `package` attribute of the `<manifest>` node: `<manifest ... package="com.companyname.myproject>"`. |
-| `ApplicationVersion` | The `android:versionCode` attribute of the `<manifest>` node: `<manifest ... android:versionCode="1">`. |
+Making your app available for direct download is most useful for a controlled subset of users, such as an enterprise environment or an app that is only meant for a small or well-specified set of users. Server and email distribution are also simpler publishing models, requiring less preparation to publish an app, though apps may be blocked as an email attachment.
 
-Here's an example of an automatically generated manifest file with the package and version information specified:
+The Amazon Mobile App Distribution Program enables mobile app developers to distribute and sell their applications on Amazon. Users can discover and shop for apps on their Android-powered devices by using the Amazon App Store application.
 
-```xml
-<manifest
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:versionCode="1"
-    package="com.hi.companyname.myproject"
-    android:versionName="1.0.0">
-    <!-- other settings -->
-</manifest>
-```
+Google Play is arguably the most comprehensive and popular marketplace for Android applications. Google Play allows users to discover, download, rate, and pay for applications by clicking a single icon either on their device or on their computer. Google Play also provides tools to help in the analysis of sales and market trends and to control which devices and users may download an application.
 
-For more information about the manifest, see [Google Android App Manifest Overview](https://developer.android.com/guide/topics/manifest/manifest-intro).
+## Related Links
```

Comment:
You used "See also" in other docs. Related links was a Xamarin thing I've tried to remove from the docs (just searched the repo and it's only in a lone doc now).

Reply:
```suggestion
## See also
```